client/my-sites/promote-post/main.tsx

```diff
@@ -174,6 +175,11 @@ export default function PromotedPosts( { tab }: Props ) {
 				<PageViewTracker path="/advertising/:site/posts" title="Advertising > Ready to Blaze" />
 			) }
 
+			<QueryStatsRecentPostViews
```

Comment:
This is used to grab the `recent views` numbers and automatic adds it to our state